examples/md-flexible/Simulation.h

```diff
@@ -0,0 +1,469 @@
+/**
+ * @file Simulation.h
+ * @author N. Fottner
+ * @date 12/05/19
+ */
+#pragma once
+
+#include <fstream>
+#include <iostream>
+
+#include "../../tests/testAutopas/testingHelpers/GaussianGenerator.h"
+#include "../../tests/testAutopas/testingHelpers/GridGenerator.h"
+#include "../../tests/testAutopas/testingHelpers/RandomGenerator.h"
```

Comment:
-> #375 but can be done later

Reply:
I think we should do this in a separate PR

Reply:
that's why I opened a new issue report :)